templates/cases/case_read.haml

```diff
@@ -169,7 +169,7 @@
                 .contact-field-label.col-sm-6
                   [[ urn.scheme ]]
                 .contact-field-value.col-sm-6
-                  [[ urn.path ]]
+                  [[ ]]
```

Comment:
At one point in time someone was using CasePro with a backend besides RapidPro and was displaying URNs. Could we maybe add a setting like `SITE_REDACT_URNS` (see other settings like `SITE_CONTACT_DISPLAY`), and then if that is set, display `••••••••` as a placeholder like we do in RapidPro?